<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateScheduleViewModal</class>
 <widget class="QWidget" name="UpdateScheduleViewModal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-weight: 600; font-size: 10pt; color: black;</string>
     </property>
     <property name="text">
      <string>Расписание</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="formLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="horizontalSpacing">
            <number>40</number>
           </property>
           <item row="1" column="1">
            <layout class="QVBoxLayout" name="inputGroup_12">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="arrDateLabel">
               <property name="text">
                <string>Время прибытия</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateTimeEdit" name="arrDateInput">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="inputGroup_16">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="depDateLabel">
               <property name="text">
                <string>Время отправления</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateTimeEdit" name="depDateInput">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0" colspan="2">
            <layout class="QVBoxLayout" name="inputGroup_6">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="routeNameLabel">
               <property name="text">
                <string>Название маршрута</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="routeNameComboBox"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="inputGroup_7">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="trainNameLabel">
               <property name="text">
                <string>Название поезда</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="trainNameComboBox"/>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QVBoxLayout" name="inputGroup_11">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="firstClassPriceLabel">
               <property name="text">
                <string>Стоимость первого класса</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="firstClassPriceInput">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>20000</number>
               </property>
               <property name="singleStep">
                <number>100</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QVBoxLayout" name="inputGroup_10">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="trainTypeLabel">
               <property name="text">
                <string>Тип поезда</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="trainTypeComboBox">
               <item>
                <property name="text">
                 <string>пассажирский</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>скорый-пассажирский</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>скоростной-пассажирский</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <layout class="QVBoxLayout" name="inputGroup_13">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="secondPriceClassLabel">
               <property name="text">
                <string>Стоимость второго класса</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="secondPriceClassInput">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>20000</number>
               </property>
               <property name="singleStep">
                <number>100</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveChangesButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Сохранить изменения</string>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
